share/smack/top.py

```diff
@@ -175,6 +175,9 @@ def arguments():
   translate_group.add_argument('--float', action="store_true", default=False,
     help='enable bit-precise floating-point functions')
 
+  translate_group.add_argument('--split-structs', action='store_true', default=False,
```

Comment:
Do we really want this to be optional? Are there any bad things that can happen by splitting structs into individual accesses? Clearly not doing it is less precise.